doc/index.txt

```diff
@@ -21,6 +21,9 @@ learning/machine learning <https://mila.umontreal.ca/en/cours/>`_ classes).
 News
 ====
 
+* Removed support for the old (device=gpu) backend.  Use the new
+  backend (device=cuda) for gpu computing.
+
```

Comment:
It would make porting the doc harder. What about making it clear on the line that this is in the dev version so we can keep the same file?